.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+per_file_ignores = __init__.py:F401, settings.py:E501
```

Comment:
I think you can add all the flake8 config in pyproject.toml

Reply:
I tried to do it before (including it in pyproject.toml) but it either does not work or throws many errors.